src/daft-sql/src/planner.rs

```diff
@@ -822,18 +822,18 @@ impl<'a> SQLPlanner<'a> {
                     self.plan_relation_table(name)?
                 };
 
-                // we require all logical plans to be named, apply the name from the identifier used to resolve this relation.
-                let alias = if let Some(alias) = alias {
-                    Some(alias.clone())
-                } else {
+                // we require all logical plans to be named, apply the name from the identifier used to resolve this table.
```

Comment:
Why do all logical plans need to be named? I may be wrong, but I believe if you pass in a path (such as `FROM 'file.parquet'`), I wouldn't expect you to be able to access it by the name "file" unless it's explicitly set, such as `SELECT file.* FROM 'file.parquet' AS file`.

Reply:
i believe the only place where the table will implicitly have a name when we get it from `SQLPlanner::plan_relation_table`. Since we don't do aliasing in the session anymore, we could maybe do it in `SQLPlanner::get_table`?

Reply:
That makes sense, I can update this to do so. This is just a simple rewrite to derive an alias from the table name when not give.

```
FROM <cat>.<schema>.<table>

-- to

FROM <cat>.<schema>.<table> AS <table>
```